### ISSUE TITLE
🐛 Bug Report: 

### ISSUE THREAD

**Issue by UseLEss213** [appwrite/appwrite#7039]
### 👟 Reproduction steps

1. open the appwrite dashboard and navigate to the appropriate collection and document.
2. the document contains an array field with identical values for each element of the array (i.e., ["test", "test", "test"]).
3. attempt to delete one or more elements from this array field.
Notice that the "Update" button does not become enabled, preventing the changes from being saved.

### 👍 Expected behavior

The "Update" button should become enabled as soon as any changes are detected in the document, allowing me to save my changes and successfully delete elements from the array.

### 👎 Actual Behavior

The "Update" button remains disabled and i am unable to save my changes. t
![Screenshot 2023-10-27 182518](https://github.com/appwrite/sdk-for-python/assets/134964492/cbc8d985-fbcf-46e7-9817-8e9856442031)
![Screenshot 2023-10-27 182620](https://github.com/appwrite/sdk-for-python/assets/134964492/cff11197-6b6e-4c54-88e6-a42719092063)
his seems to occur when all elements in the array are identical.

### 🎲 Appwrite version

Version 0.10.x

### 💻 Operating system

Windows

### 🧱 Your Environment

no

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/master/CODE_OF_CONDUCT.md)

**Comment by shashankdaima** [comment]
I would like to work on this? can I assigned to this?

**Comment by nick2432** [comment]
can i work on this?

**Comment by stnguyen90** [comment]
@shashankdaima, would you still like to work on this?

**Comment by shashankdaima** [comment]
Yes, I would love to, but will need some of your's (and other appwrite members') assistance. Should I ask those question here? or on discord? Any particular discord channel for contribution?

**Comment by stnguyen90** [comment]
@shashankdaima, assigned! It would be best to create a support post.

**Comment by stnguyen90** [comment]
Unassigning due to inactivity
